### PR TITLE
Simplify Edge Case Handling for identifiers in Delta and Deleted Directory

### PR DESCRIPTION
**Pull request by georgend**
- Add Data Scripts for Deleted Directory Object. 
- Swap out incorrect known placeholder for a known placeholder that exists in identifiers.json.
- Correct the same logic for delta.
- Update delta scripts.
- Remove known issues.

### CODE REVIEW

File: scripts/TenantSetup/TenantHttpData/Application/Application.ps1 [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+. $raptorUtils
+
+$appSettings = Get-AppSettings
+$identifiers = Get-CurrentIdentifiers -IdentifiersPath $IdentifiersPath
+$domain = Get-CurrentDomain -AppSettings $appSettings
+
+#Connect To Microsoft Graph Using ClientId, TenantId and Certificate in AppSettings
+Connect-DefaultTenant -AppSettings $appSettings
+
+#Create Application to be Deleted  https://docs.microsoft.com/en-us/graph/api/activitybasedtimeoutpolicy-post-activitybasedtimeoutpolicies?view=graph-rest-1.0&tabs=http
```
Comment (zengin) [inline scripts/TenantSetup/TenantHttpData/Application/Application.ps1:15]: linked documentation looks unrelated.

File: msgraph-sdk-raptor-compiler-lib/Models/IdentifierReplacer.cs [hunk 2]
```diff
@@ -87,8 +87,13 @@ private string ReplaceEdgeCases(string input)
                 // https://docs.microsoft.com/en-us/graph/api/event-delta?view=graph-rest-1.0&tabs=csharp
                 { "new QueryOption(\"startdatetime\", \"{start_datetime}\"", $"new QueryOption(\"startdatetime\", \"{yesterday}\"" },
                 { "new QueryOption(\"enddatetime\", \"{end_datetime}\"", $"new QueryOption(\"enddatetime\", \"{now}\"" },
```
Comment (zengin) [inline msgraph-sdk-raptor-compiler-lib/Models/IdentifierReplacer.cs:89]: please convert method back to static.

File: scripts/TenantSetup/TenantHttpData/Application/Application.ps1 (hunk 1 above)
Comment (georgend) [reply scripts/TenantSetup/TenantHttpData/Application/Application.ps1:15]: My bad.

**Review by zengin — APPROVED** [review]
